### ISSUE TITLE
Dataset metadata

### ISSUE THREAD

**Issue by SkepticRaven** [DeepLabCut/benchmark#7]
The datasets don't contain much documentation surrounding how the annotations were generated. There's only a handful of sentences in the paper describing these.

I'm particularly interested about annotation, for example (quote from paper for the tri-mouse dataset):
> One human annotator was instructed to localize the 12 keypoints (snout, left ear, right ear, shoulder, four spine points, tail base and three tail points) across 161 frames sampled from within DeepLabCut using the k-means clustering approach (across eight videos).

Since the training data released contains 8 video folders, does this mean that frames in the test dataset had other frames in the video annotated? If so, how far separated in time were they? How long were the videos that were sampled from? Where is k-means clustering sampling described (what parameters were used)?

It would also be great to get a datasheet for these datasets. (https://arxiv.org/pdf/1803.09010.pdf)

**Comment by MMathisLab** [comment]
@SkepticRaven - the `kmeans` parameters were the default parameters in DeepLabCut `extract_frames`. Yes, the train and test set was across the videos, again using default parameters in the code in `create_training_dataset`. The image index ID is in the data format so you can reconstruct the differences, but it was not systematically selected to be close or near ... 

https://benchmark.deeplabcut.org/
